lib/db_utils.go

```diff
@@ -587,7 +587,11 @@ type DBPrefixes struct {
 	// Prefix, <ProfilePKID [33]byte>, <LockupDurationNanoSecs int64> -> <LockupYieldCurvePoint>
 	PrefixLockupYieldCurvePointByProfilePKIDAndDurationNanoSecs []byte `prefix_id:"[94]" is_state:"true"`
 
-	// NEXT_TAG: 95
+	// PrefixValidatorBLSPublicKeyToBLSPublicKeyPKIDPairEntry: Retrieve a BLSPublicKeyPKIDPairEntry by BLS public key.
+	// Prefix, <BLSPublicKey [33]byte> -> *BLSPublicKeyPKIDPairEntry
+	PrefixValidatorBLSPublicKeyToBLSPublicKeyPKIDPairEntry []byte `prefix_id:"[95]" is_state:"true"`
```

Comment:
Why not just name this `PrefixValidatorBLSPublicKeyPKIDPairEntry`. No need to repeat the BLSPublicKey part. IMO it's preferable to be concise and clear than to have the name be the most technically correct
```suggestion
	PrefixValidatorBLSPublicKeyPKIDPairEntry []byte `prefix_id:"[95]" is_state:"true"`
```